contrib/python-bindings/source/cell_accessor_wrapper.cc

```diff
@@ -1,6 +1,6 @@
 // ---------------------------------------------------------------------
 //
-// Copyright (C) 2016 by the deal.II authors
+// Copyright (C) 2016 - 2017 by the deal.II authors
```

Comment:
It would be nice to commit this, run the script again, and then squash so that these all say 2018.